### PR TITLE
Wiki reorg: Remove RPC calls from Build and Maintain

### PR DESCRIPTION
**Pull request by filippoweb3**
Transfer to Chain State Values page

### CODE REVIEW

File: docs/maintain/maintain-guides-how-to-chill.md [hunk 1]
```diff
@@ -75,9 +75,7 @@ requirements.
 :::note Chill Threshold
 
 `ChillThreshold` defines how close to the max nominators or validators we must reach before users
-can start chilling one-another. This value is currently set to
-{{ polkadot: <RPC network="polkadot" path="query.staking.chillThreshold" defaultValue={90}/>% :polkadot }}
-{{ kusama: <RPC network="kusama" path="query.staking.chillThreshold" defaultValue={90}/>% :kusama }}
+can start chilling one-another.
```
Comment (CrackTheCode016) [inline docs/maintain/maintain-guides-how-to-chill.md]:
```suggestion
can start chilling one another.
```

File: docs/general/chain-state-values.md [hunk 2]
```diff
@@ -63,6 +63,10 @@ The number of Polkadot validators in the active set is <RPC network="polkadot" p
 
 The auction ending period on Polkadot is <RPC network="polkadot" path="consts.auctions.endingPeriod" defaultValue={72000} filter="blocksToDays"/> days long.
 
+#### Block Hash Count
+
+On Polkadot, the maximum block number to block hash mappings to keep is <RPC network="polkadot" path="consts.system.blockHashCount" defaultValue={4096}/> (about seven hours).
```
Comment (profradha) [inline docs/general/chain-state-values.md]:
```suggestion
On Polkadot, the maximum number of block hashes retained on-chain at any given time is <RPC network="polkadot" path="consts.system.blockHashCount" defaultValue={4096}/> (which maps to seven hours given 6-second block times).
```

File: docs/general/chain-state-values.md [hunk 3]
```diff
@@ -250,6 +271,10 @@ A Kusama bounty has a predetermined duration of <RPC network="kusama" path="cons
 
 The waiting time before claiming a Kusama child bounty reward is <RPC network="kusama" path="consts.bounties.bountyDepositPayoutDelay" defaultValue={57600} filter="blocksToDays"/> days.
 
+#### Chill Threshold
```
Comment (CrackTheCode016) [inline docs/general/chain-state-values.md]: Unless I am missing something, it seems that some parameters are only Kusama or only Polkadot, when it seems a tabbed layout would be needed here to include parameters from both chains (assuming it's a general page?)

File: docs/general/chain-state-values.md [hunk 4]
```diff
@@ -79,6 +83,10 @@ A Polkadot bounty has a predetermined duration of <RPC network="polkadot" path="
 
 The waiting time before claiming a Polkadot child bounty reward is <RPC network="polkadot" path="consts.bounties.bountyDepositPayoutDelay" defaultValue={115200} filter="blocksToDays"/> days.
 
+#### Chill Threshold
```
Comment (profradha) [inline docs/general/chain-state-values.md]: Not sure if this is still relevant. Check the last info note here https://wiki.polkadot.network/docs/maintain-guides-how-to-chill#chill-other

File: docs/general/chain-state-values.md [hunk 5]
```diff
@@ -127,6 +135,15 @@ The minimum bond to nominate on Polkadot is <RPC network="polkadot" path="query.
 
 The minimum amount to participate to a Polkadot crowdloan is <RPC network="polkadot" path="consts.crowdloan.minContribution" defaultValue={50000000000} filter="humanReadable"/>.
 
+#### Minimum Validator Bond
+
+To start a validator instance on Polkadot, the
```
Comment (profradha) [inline docs/general/chain-state-values.md:136]: Can be misleading to new entrants into the ecosystem. May be add a line saying this does not guarantee entry into active set and earning rewards and link to https://wiki.polkadot.network/docs/maintain-guides-how-to-validate-polkadot#how-many-dot-do-i-need-to-become-an-active-validator

**Pull request merged**